webapp/src/app/course-welcome/course-welcome.component.ts

```
@@ -140,6 +201,200 @@
+    // Apply changes from editableCourse to selectedCourse
+    this.isEditing =
+      this.isEditingName =
+      this.isEditingDescription =
+      this.isEditingImage =
+        false;
+
+    this.selectedCourse.name = this.courseName;
+    this.selectedCourse.description = this.courseDescription;
+
+    if (this.chosenFile) {
+      let imageName = this.setCourseIamge(this.selectedCourse);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'imageName' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)